Estib/s3 archives tags not included (#4520)
DataDog/documentation: content/en/logs/archives/s3.md

}
```

> **Note:** At this time, archives do not include log tags (which are metadata about logs that help to connect your log data to related metrics and traces), but instead only log content. This includes the message, custom attributes, and reserved attributes of your log events.

Reviewer: ```suggestion
**Note:** Archives only include log content which consists of the message, custom attributes, and reserved attributes of your log events. The log tags (metadata that connects your log data to related metrics and traces) are not included.
```
